### ISSUE TITLE
question about the object detection code open source

### ISSUE THREAD

**Issue by challow0** [JinyuanLiu-CV/TarDAL#2]
Thanks for your great work ~  I found the code doesn't involve the cooperative training code (TarDAL with YOLOv5). Do you have a plan to open source the whole code in the future?

**Comment by MisakiCoca** [comment]
Thank you for the heads-up.
I'll collate the cooperative training code and upload it soon I finish my current project.
It won't take long, probably within a month.

**Comment by challow0** [comment]
Good~ Thanks for your reply. 

:)

**Comment by MisakiCoca** [comment]
This issue can be closed after we upload these code. :)

**Comment by MisakiCoca** [comment]
This issue can be closed after we upload these code. :)

**Comment by zugofn** [comment]
Excuse me！
How to split the validation set and training set in object detection, in the paper?
thank you🙂

**Comment by MisakiCoca** [comment]
The vast majority of the code currently in the repository is only a transitional version, and we will release the full version with joint loss of object detection within a month. @challow0

**Comment by challow0** [comment]
> The vast majority of the code currently in the repository is only a transitional version, and we will release the full version with joint loss of object detection within a month. @challow0

Alright ~, Hope you can release the code soon. Because it has been passed more than one month since the last time you say. 😂
